### DISCUSSION TITLE
Set artist and album field for notification

### DISCUSSION THREAD

**Discussion by soramister** [mediagrid/capacitor-native-audio#18] [2. Feature Ideas]
Hello,

Is it possible to add the following fields for artist and album in the notification? It would be like in this plugin : 
https://github.com/jofr/capacitor-media-session

It is supported only the title and artwork right now. I don't know if it is difficult to implement.

**Comment by wsamoht** [comment]
It should be fairly easy to add. It's not an option right now because we didn't need it when developing  the plugin :)

**Comment by wsamoht** [comment]
This has been completed in https://github.com/mediagrid/capacitor-native-audio/pull/24.

I am not sure where the album title shows as I don't see it in either the Android or iOS notification.

**Reply by wsamoht** [reply]
Also, when calling `changeMetadata()` any params not sent will now explicitly be set to an empty value. Before we didn't touch them. It's simpler logic and I think makes more sense. This could be considered a breaking change but don't think it warrants a major version bump.

**Reply by jeremieoneglancestudio** [reply]
Thank you for this modification !

The album title, when provided will add the information with the rest of the track. An example below on car play.
![thread-229290230-2005590456062826249](https://github.com/user-attachments/assets/5676ccc9-7bcc-4e75-951f-3459daa54fd6)

**Reply by jeremieoneglancestudio** [reply]
> Also, when calling `changeMetadata()` any params not sent will now explicitly be set to an empty value. Before we didn't touch them. It's simpler logic and I think makes more sense. This could be considered a breaking change but don't think it warrants a major version bump.

Make sense to me.
